git/tagging-versions.md

```diff
@@ -0,0 +1,108 @@
+# Tagging versions
+
+Or, how to capture relevant information about deployments.
+
+<a name="rationale"></a>
+## 0. Rationale
+
+Without a plan, production deployments will happen and information about what
+was deployed will be scattered in various places. Hotfixes can add more
+complexity and confusion. The need almost invariably arises for a specific
+accounting of what was deployed. To that end, I propose the following best
+practice.
+
+<a name="levels"></a>
+## 1. Levels of information
+
+Git already records commits, which include a summary message by the developer
+making the change. This level of information—both the actual files and lines
+changed as well as the summary messages—are the ground-level truth of what
+changed. It should be possible to easily list the commits specific to a version.
+
+The customer typically does not care about this level of information. Instead,
+customers track things at the level of tasks and features. Such features are
+typically tracked in Trello or another project management tool as "cards" (or
+similar). Cards have an associated identifying number. These card numbers should
+be explicitly listed in the information about a deployment using the process
+described below.
+
+<a name="versions"></a>
+## 2. Versions
+
+Each deployment should have a version number associated with it. There is a
+one-to-one relationship between deployments and versions. Each version number is
+in the form `major.minor.patch`, e.g. 2.4.1, where the patch is optional and, if
+not specified, defined to be 0 (so `2.4` is a sort of shorthand for `2.4.0`).
```

Comment:
One awkward part of this PR is that different projects have different needs. For example, most of my projects follow a versioning scheme of `v2016-12-15`, and sometimes add an extra digit to the end when there are multiple deploys on a single day: `v2016-12-15.2`. Seems to work well for support projects.

For support projects that don't have formal release requirements, trying to determine major/minor/patch versions didn't seem worth the mental overhead, especially since our projects often don't map well to with the semantic versioning rules. 

I think both versioning schemes are perfectly fine, so I recommend at least noting the versioning schema is a project dependent decision, and not make semantic-like versioning the "RoleModel Way".